guides/source/active_record_validations.md

```diff
@@ -119,46 +119,76 @@ careful.
 The following methods trigger validations, and will save the object to the
 database only if the object is valid:
 
-* `create`
-* `create!`
-* `save`
-* `save!`
-* `update`
-* `update!`
+* [`create`][]
+* [`create!`][]
+* [`save`][]
+* [`save!`][]
+* [`update`][]
+* [`update!`][]
 
 The bang versions (e.g. `save!`) raise an exception if the record is invalid.
 The non-bang versions don't: `save` and `update` return `false`, and
 `create` returns the object.
 
+[`create`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-create
+[`create!`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-create-21
+[`save`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save
+[`save!`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save-21
+[`update`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update
+[`update!`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update-21
+
 ### Skipping Validations
 
 The following methods skip validations, and will save the object to the
 database regardless of its validity. They should be used with caution.
 
-* `decrement!`
-* `decrement_counter`
-* `increment!`
-* `increment_counter`
-* `insert`
-* `insert!`
-* `insert_all`
-* `insert_all!`
-* `toggle!`
-* `touch`
-* `touch_all`
-* `update_all`
-* `update_attribute`
-* `update_column`
-* `update_columns`
-* `update_counters`
-* `upsert`
-* `upsert_all`
+* [`decrement!`][]
+* [`decrement_counter`][]
+* [`increment!`][]
+* [`increment_counter`][]
+* [`insert`][]
+* [`insert!`][]
+* [`insert_all`][]
+* [`insert_all!`][]
+* [`toggle!`][]
+* [`touch`][]
+* [`touch_all`][]
+* [`update_all`][]
+* [`update_attribute`][]
+* [`update_attribute!`][]
+* [`update_column`][]
+* [`update_columns`][]
+* [`update_counters`][]
+* [`upsert`][]
+* [`upsert_all`][]
 
 Note that `save` also has the ability to skip validations if passed `validate:
 false` as an argument. This technique should be used with caution.
 
 * `save(validate: false)`
 
+WARNING. Some of these methods may also skip callbacks. Please check their documentation to learn more.
```

Comment:
@Earlopain i think hyperlinking to the method's documentation should be enough. for a learner who's as yet unfamiliar with callbacks but trying to figure out how to skip validations (or whatever got them to this page), a new concept/direction (aka callbacks) could confuse instead of clarify. imo, the other effects/abilities of the method is best left to the documentation.

Reply:
Yeah, I can see how that might be undesirable. I've instead added a sentence to check out out the method docs instead to both guides. Do you think it's better this way? I'm not that good with words, if you've got suggestions to make it sound better I'm all ears.

Reply:
yup, looks good to me! thank you ❤️